samples/crypto/hkdf/README.rst

```diff
@@ -7,20 +7,23 @@ Crypto: HKDF
    :local:
    :depth: 2
 
+The HMAC Key Derivation Function (HKDF) sample shows how to derive keys using a sample key, salt, and additional data.
```

Comment:
"derive keys with the HKDF algorithm, using sample key, salt, and additional data"

Reply:
I don't mind to do that but do we not write HKDF twice in a single sentence then?

Reply:
No, I don't have a problem with the duplication.